process.py

```diff
@@ -386,9 +387,15 @@ def get_provider_code(directory_name=None):
 
 def main():
     parser = argparse.ArgumentParser()
-    parser.add_argument('--path', type=str, default="static/data/lms")
+    specific_folder = raw_input('Do you want to load a specific folder? type y to load a specific folder, type n to load all the data\n')
```

Comment:
I actually liked that everything is controlled from the command line options, without interactive input. It's a bit confusing if there are both. Also, now you can't just run the script, you have to type `n` to get the old behavior...